drivers/bridge/bridge_store.go

```diff
@@ -185,6 +186,7 @@ func (ncfg *networkConfiguration) UnmarshalJSON(b []byte) error {
 	ncfg.BridgeName = nMap["BridgeName"].(string)
 	ncfg.EnableIPv6 = nMap["EnableIPv6"].(bool)
 	ncfg.EnableIPMasquerade = nMap["EnableIPMasquerade"].(bool)
+	ncfg.NDPProxyInterface = nMap["NDPProxyInterface"].(string)
```

Comment:
Given this new field is not there for existing bridge network in store, you need to first check if the entry exists in the map before converting to string, or it will panic.